docs/literate.jl

```diff
@@ -39,8 +46,24 @@ function preprocess(content)
 
     # remove VSCode `##` block delimiter lines
     content = replace(content, r"^##$."ms => "")
-
-    return content
+    # adds the current version of the packages
+    append = """
+    # ### Package information
+    # #### Package version
+    # ```julia
+    $(chomp(replace(pkg_status, r"^"m => "# ")))
+    # ```
+    # #### Manifest
+    # <details>
```

Comment:
The preview indicates that this does not work as intended with Documenter. Maybe putting it in a "```@raw html" block (https://juliadocs.github.io/Documenter.jl/stable/man/syntax/#@raw-format-block) would fix it?

Reply:
In the notebook version the HTML summary seems to work but the Julia block is not shown correctly: https://nbviewer.org/github/JuliaGaussianProcesses/ApproximateGPs.jl/blob/gh-pages/previews/PR97/examples/a-regression.ipynb